packages/plugin-compiler/src/main/kotlin/io/realm/compiler/Identifiers.kt

```diff
@@ -54,10 +54,10 @@ internal object Names {
 
 internal object FqNames {
     // TODO we can replace with RealmObject::class.java.canonicalName if we make the runtime_api available as a compile time only dependency for the compiler-plugin
-    val REALM_OBJECT_ANNOTATION = FqName("io.realm.runtimeapi.RealmObject")
     val REALM_NATIVE_POINTER = FqName("io.realm.runtimeapi.NativePointer")
     val REALM_MODULE_ANNOTATION = FqName("io.realm.runtimeapi.RealmModule")
-    val REALM_MODEL_INTERFACE = FqName("io.realm.runtimeapi.RealmModelInternal")
+    val REALM_MODEL_INTERNAL_INTERFACE = FqName("io.realm.runtimeapi.RealmModelInternal")
```

Comment:
Maybe cleaning up the namespace would render us able to reason about what is what and we could eliminate the `_INTERFACE`?